src/loader/loader.cr

```diff
@@ -69,7 +69,7 @@ module BakedFileSystem
       def write(slice : Bytes)
         slice.each do |byte|
           case byte
-          when 35_u8..91_u8, 93_u8..127_u8
+          when 36_u8..91_u8, 93_u8..127_u8
```

Comment:
Maybe add `when 35_u8; @io << "\\#"`? That would result in a few bytes less in the baked string.